client/src/js/app.js

```diff
@@ -218,6 +218,11 @@ function bhimaConfig($stateProvider, $urlRouterProvider, $urlMatcherFactoryProvi
     controller: 'ComplexJournalVoucherController as ComplexVoucherCtrl',
     templateUrl: 'partials/vouchers/complex.html'
   })
+  .state('vouchersRecords', {
+    url : '/vouchers/records',
```

Comment:
This url should just be `/vouchers`.  For reference:
 1. `/invoices` -> invoice registry
 2. `/patients` -> patient registry

@sfount proposed this model in #391.